### PR TITLE
Search and sort users with ransack

### PR DESCRIPTION
**Pull request by oleg08**
code review:
- [ ] @CHILL99 
- [ ] @krisberry 
- [ ] @natalya5 
- [ ] @DimaSmagulov 
- [x] @OrestF 
- [ ] @rtershak 
- [x] @vvs79 

2nd lvl review:

- [x] @Kmalynss
- [ ] @serhiym 
- [ ] @ruba-ruba

https://trello.com/c/Pzln1WRG/41-as-an-admin-when-on-users-index-i-want-to-be-able-to-search-for-user

### CODE REVIEW

File: app/views/admin/users/_search_form_for.html.slim [hunk 1]
```diff
@@ -0,0 +1,5 @@
+= search_form_for @search, url: admin_users_path do |f|
+  .field
+    = f.label :first_name_cont, "First Name contains"
```
Comment (DimaSmagulov) [inline app/views/admin/users/_search_form_for.html.slim]: try to doit with bootstrap
<div class="col-lg-6">
    <div class="input-group">
      <input type="text" class="form-control" placeholder="Search for...">
      <span class="input-group-btn">
        <button class="btn btn-default" type="button">Go!</button>
      </span>
    </div>
  </div>

File: app/views/admin/users/_search_form_for.html.slim [hunk 2]
```diff
@@ -0,0 +1,5 @@
+= search_form_for @search, url: admin_users_path do |f|
+  .field
```
Comment (DimaSmagulov) [inline app/views/admin/users/_search_form_for.html.slim]: better use bootstrap
smth like this 
.col-lg-6
  form.input-group
    = search_form_for @search, url: admin_users_path do |f|
      = f.text_field :first_name_cont, :placeholder => "Search for...", class: "form-control"
      span.input-group-btn
        = f.submit "Go!", class: 'btn btn-default type="button"'

File: Gemfile.lock [hunk 3]
```diff
@@ -39,12 +39,12 @@ GEM
       tzinfo (~> 1.1)
     ansi (1.5.0)
     arel (6.0.3)
-    aws-sdk (2.2.6)
-      aws-sdk-resources (= 2.2.6)
-    aws-sdk-core (2.2.6)
+    aws-sdk (2.2.7)
+      aws-sdk-resources (= 2.2.7)
+    aws-sdk-core (2.2.7)
       jmespath (~> 1.0)
```
Comment (OrestF) [inline Gemfile.lock]: We need to use aws-sdk v 1.6.0 in other way we became an error when we try to upload a picture.
I fix this in my PR but if yours pass first we became an error on heroku.

**Comment by oleg08** [comment]
Dima and Orestes notes are addressed and the changes already made.
Rebase is done

**Comment by serhiym** [comment]
@oleg08 - looks good
just one note - show some message when no results found

**Comment by vvs79** [comment]
reviewed

File: app/views/admin/users/index.html.slim [hunk 4]
```diff
@@ -9,18 +8,23 @@
...
       th width="30%"
         h4 Email
       th width="5%"
         h4 Approved
       th width="5%"
         h4 Admin
+    - if @users.empty?
+      .alert.alert-warning.not_found role="alert" users not found
+    - else
     = render @users
```
Comment (Kmalynss) [inline app/views/admin/users/index.html.slim]: ``` render @users``` must be inside ```else```

**Comment by Kmalynss** [comment]
gtg

**Pull request merged**